.github/workflows/reusable-prober.yml

```diff
@@ -167,7 +167,6 @@ jobs:
         if: success() || failure()
         run: |
           echo "root_state=good" >> $GITHUB_OUTPUT
```

Comment:
Instead of removing this line, should we just comment it out, add another comment pointing out to the issue tracking the failure, that we can then keep track of to make sure to revert next week?

Reply:
I just saw the comment from @haydentherapper so it's fine to go in as is from my POV, the point just being that there's enough tracking to make sure that we can easily revert the change. I'm not assuming there will be enough changes to cause confusion between now and revert however, so let's just merge as is.

Reply:
I've filed an [issue](https://github.com/sigstore/sigstore-probers/issues/112) and will add this to the on-call handoff notes for next week so we don't lose track of this.